hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -323,22 +324,22 @@ def get_signals() -> t.List[SignalSourceSummary]:
     TODO this should be updated to check ThreatExchangeConfig
     based on what it finds in the config it should then do a s3 select on the files
     """
-    mocked_signal_type1a = SignalSourceType(type="HASH_PDQ", count=12456)
-    mocked_signal_type1b = SignalSourceType(type="MOCKED_TYPE1", count=456)
-    mocked_signal_type2a = SignalSourceType(type="HASH_PDQ", count=2456)
-    mocked_signal_type2b = SignalSourceType(type="MOCKED_TYPE2", count=956)
-
-    mocked_signal_source1 = SignalSourceSummary(
-        name="Mocked Signal Source 1",
-        signals=[mocked_signal_type1a, mocked_signal_type1b],
-        updated_at="mocked datetime1",
-    )
-    mocked_signal_source2 = SignalSourceSummary(
-        name="Mocked Signal Source 2",
-        signals=[mocked_signal_type2a, mocked_signal_type2b],
-        updated_at="mocked datetime2",
-    )
-    return [mocked_signal_source1, mocked_signal_source2]
+    signals = []
+    counts = get_signal_hash_count()
+    for dataset, total in counts.items():
+        if dataset.endswith(THREAT_EXCHANGE_PDQ_FILE_EXTENSION):
+            dataset_name = dataset.replace(
```

Comment:
So for now dataset_name will always be the collaboration ID?

Reply:
yeah, the <filename> of <filename>.pdq.te is which for now is the pg id